Neos.ContentRepository/Migrations/Mysql/Version20190905074145.php

```diff
@@ -0,0 +1,41 @@
+<?php
+namespace Neos\Flow\Persistence\Doctrine\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Create index on path for nodedata table
+ */
+class Version20190905074145 extends AbstractMigration
+{
+    /**
+     * @return string
+     */
+    public function getDescription()
+    {
+        return 'Create index on path for nodedata table';
+    }
+
+    /**
+     * @param Schema $schema
+     * @return void
+     */
+    public function up(Schema $schema)
+    {
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on "mysql".');
+
+        $this->addSql('CREATE INDEX path ON neos_contentrepository_domain_model_nodedata (path(255))');
```

Comment:
Ahm, no. It should have been called `pathindex`…

Reply:
Arrgh, I think I confused myself somehow. Should I create a new PR to fix this? I assume this will be dropped with with the new CR? If this should be changed, @kdambekalns, shouldn't the `parentpath` index also be renamed? The easier fix might be just to change `pathindex` to `path` in `NodeData.php` to be consistent with `parentpath`.